packages/mobile/src/screens/track-screen/TrackScreen.tsx

```diff
@@ -152,7 +149,6 @@ export const TrackScreen = () => {
         lineup={lineup}
         start={1}
         includeLineupStatus
-        selfLoad
```

Comment:
does this impact scrolling pagination behavior? (i guess not necessary for track screen lineup anyway)

Reply:
yeah exactly, we limit to 6 so shouldnt ever need to "loadMore", and the first 6 tracks get fetched in the `fetchTrack` saga chain. this was actually leading to a runtime error since it was fetching, but didnt have the right data, since it needs ownerHandle, etc which are calculated and used in saga's call